src/css/index.css

```diff
@@ -0,0 +1,535 @@
+    opacity: 40%;
+}
+
+.banner__content-item:nth-child(3) .banner__content-background {
+    opacity: 40%;
+}
+
+.banner__content-item:nth-child(4) .banner__content-background {
+    opacity: 10%;
+}
+
+.banner__content-item:nth-child(4) .banner__content-text {
+    background-color: var(--button-bg-color);
+    color: white;
+}
+
+.banner__content-text {
+    position: absolute;
+    padding: 10px;
+    background-color: var(--banner-bg-color);
+    font-size: var(--h3-font);
+    word-wrap: break-spaces;
+}
+
+/*##title*/
+
+.title {
+    margin-top: var(--top-margin);
+    border-bottom: 1px solid var(--title-border-color);
+}
+
+.title__content {
+    display: flex;
+    flex-wrap: wrap;
+}
+
+.title__content-text {
+    width: 80%;
+}
+
+.title__content-main-text {
+    font-size: var(--h1-font);
+    color: var(--main-text-color);
+}
+
+.title__content-subtext {
+    font-size: var(--h3-font);
+    color: var(--sub-text-color);
+}
+
+.title__content-img {
+    font-size: var(--h2-font);
+    color: var(--button-fg-color);
+}
+
+.title__content-button {
+    margin-bottom: 65px;
+    width: 20%;
+    padding: 15px;
+    font-size: var(--h2-font);
+}
+
+/*##row*/
+
+.row {
+    margin-top: var(--top-margin);
+}
+
+.row__content {
+    display: flex;
+    flex-flow: row;
+    flex-wrap: wrap;
+    margin-top: 10px;
+    width: 100%;
+}
+
+.row__content-item {
+    display: flex;
+    justify-content: left;
+    align-items: center;
+    width: 50%;
+    margin: 10px auto;
+}
+
+/*rounded images*/
+.row__content-span {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    border: 2px solid var(--row-circle-color);
+    border-radius: 130px;
+    min-height: 80px;
+    min-width: 80px;
+    width: 8vw;
+    height: 8vw;
+    text-align: center;
+}
+
+.row__content-img {
+    display: inline;
+    font-size: 40px;
+    color: var(--content-img-color);
+}
+
+.row__content-article {
+    margin: 5%;
+}
+
+.row__content-title {
+    font-size: var(--h2-font);
+    color: var(--main-text-color);
+}
+
+.row__content-text {
+    margin-top: 10px;
+    font-family: var(--sub-font);
+    font-size: var(--p-link-font);
+    color: var(--sub-text-color);
+}
+
+/*##subscribe*/
+
+.subscribe {
+    margin-top: var(--top-margin);
+}
+
+.subscribe__content {
+    display: flex;
+    align-items: center;
+    flex-wrap: wrap;
+    border-radius: 5px;
+    height: 100px;
+    background: var(--subscribe-bg-color);
+}
+
+.subscribe__content-text {
+    width: 25%;
+    text-align: center;
+    font-size: var(--h1-font);
+    color: var(--subscribe-title-color);
+}
+
+.subscribe__content-form {
+    display: flex;
+    justify-content: space-around;
+    flex-wrap: wrap;
+    margin-right: 10px;
+    width: 70%;
+}
+
+.subscribe__content-input {
+    border: none;
+    border-radius: 2px;
+    width: 33%;
+    padding: 10px 15px;
+    font-family: var(--sub-font);
+    font-size: var(--p-link-font);
+}
+
+.subscribe__content-button {
```

Comment:
you better not to link naming to the context as well as styling. try to find more general and abstract name. most of the orange buttons on the page are more similar than they share in styling for now. `button button--small` or something as an example

Reply:
But every similar style property, like orange background, is defined in general ".button" style. I added inherited font-family as a general button style, now it should look more similar. But if we speak about font-size and padding, they are different, according to PSD-specification. That's why i decided to use context-specific name. So maybe it can stay like it is?

Reply:
What I meant is that this kind of approach makes css not scalable. Of course, it can stay like this and usually not a big problem for a small project. But what if I'll need another button which is the same as subscribe button but in another context? will you copy the name? so you'll need to pull the context there, refactor the code (also thinking about not to break all places where that class was used), rename buttons or copy styling... or anything else.

So the main idea is how to deal with big projects on the small tasks.